.golangci.yml

```diff
@@ -0,0 +1,32 @@
+linters-settings:
```

Comment:
I would suggest in adding golangci-lint as build target / verification script somewhere inside the CI.